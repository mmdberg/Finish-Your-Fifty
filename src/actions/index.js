export const captureUser = (user) => ({
  type: 'CAPTURE_USER',
  user
});

export const addRace = (race) => ({
  type: 'ADD_RACE',
  race
});

export const logOut = () => ({
  type: 'LOG_OUT'
});

export const removeRace = (race) => ({
  type: 'REMOVE_RACE',
  race
});