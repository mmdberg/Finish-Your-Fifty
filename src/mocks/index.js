export const mockRace = {
  venue: 'Atlas Park',
  city: 'San Diego',
  event: 'Spooky 5k',
  date: '04-05-2018'
};

export const mockApiResult = {
  place: {
    placeName: 'Atlas Park',
    cityName: 'San Diego'
  },
  activityStartDate: '2018-04-05T08:00:00',
  organization: {
    organizationName: 'Spooky 5k'
  }
};

export const mockUser = {
  name: 'Pizza',
  email: 'pizza@pizza',
  password: 'p'
};