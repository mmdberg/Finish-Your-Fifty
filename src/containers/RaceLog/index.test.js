import { shallow } from 'enzyme';
import { RaceLog, mapStateToProps, mapDispatchToProps } from './index';
import { mockRace, mockUser } from '../../mocks';
import React from 'react';
import * as actions from '../../actions';

describe('RaceLog', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<RaceLog races={[mockRace]}/>);
    expect(wrapper).toMatchSnapshot();
  });

  describe('mapStateToProps', () => {
    it('should correctly map races to props', () => {
      const mockState = {
        races: [mockRace]
      };
      const mapped = mapStateToProps(mockState);
      expect(mapped.races).toEqual([mockRace]);
    });

    it('should correctly map user to props', () => {
      const mockState = {
        user: mockUser
      };
      const mapped = mapStateToProps(mockState);
      expect(mapped.user).toEqual(mockUser);
    });
  });

  describe('mapDispatchToProps', () => {
    it('should call dispatch with the correct params on removeRace', () => {
      const mockDispatch = jest.fn();
      const expected = actions.removeRace(mockRace);
      const mapped = mapDispatchToProps(mockDispatch);
      mapped.removeRace(mockRace);
      expect(mockDispatch).toHaveBeenCalledWith(expected);
    });
  });
});