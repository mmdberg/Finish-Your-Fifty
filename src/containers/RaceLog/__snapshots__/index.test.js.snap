// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RaceLog should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RaceLog
    races={
        Array [
            Object {
              "city": "San Diego",
              "date": "04-05-2018",
              "id": 1577,
              "raceName": "Spooky 5k",
              "state": "CA",
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ReactTable
        AggregatedComponent={[Function]}
        ExpanderComponent={[Function]}
        FilterComponent={[Function]}
        LoadingComponent={[Function]}
        NextComponent={undefined}
        NoDataComponent={[Function]}
        PadRowComponent={[Function]}
        PaginationComponent={[Function]}
        PivotComponent={undefined}
        PivotValueComponent={[Function]}
        PreviousComponent={undefined}
        ResizerComponent={[Function]}
        TableComponent={[Function]}
        TbodyComponent={[Function]}
        TdComponent={[Function]}
        TfootComponent={[Function]}
        ThComponent={[Function]}
        TheadComponent={[Function]}
        TrComponent={[Function]}
        TrGroupComponent={[Function]}
        aggregatedKey="_aggregated"
        className=""
        collapseOnDataChange={true}
        collapseOnPageChange={true}
        collapseOnSortingChange={true}
        column={
                Object {
                        "Aggregated": undefined,
                        "Cell": undefined,
                        "Expander": undefined,
                        "Filter": undefined,
                        "Footer": undefined,
                        "Header": undefined,
                        "Pivot": undefined,
                        "PivotValue": undefined,
                        "aggregate": undefined,
                        "className": "",
                        "filterAll": false,
                        "filterMethod": undefined,
                        "filterable": undefined,
                        "footerClassName": "",
                        "footerStyle": Object {},
                        "getFooterProps": [Function],
                        "getHeaderProps": [Function],
                        "getProps": [Function],
                        "headerClassName": "",
                        "headerStyle": Object {},
                        "minWidth": 100,
                        "resizable": undefined,
                        "show": true,
                        "sortMethod": undefined,
                        "sortable": undefined,
                        "style": Object {},
                      }
        }
        columns={
                Array [
                        Object {
                          "Header": "Race",
                          "accessor": "raceName",
                        },
                        Object {
                          "Header": "Distance",
                          "accessor": "distance",
                        },
                        Object {
                          "Header": "Time",
                          "accessor": "time",
                        },
                        Object {
                          "Header": "City",
                          "accessor": "city",
                        },
                        Object {
                          "Header": "State",
                          "accessor": "state",
                        },
                        Object {
                          "Header": "Date",
                          "accessor": "date",
                        },
                        Object {
                          "Header": "Completed?",
                          "accessor": "completed",
                        },
                        Object {
                          "Header": "Remove",
                          "accessor": "remove",
                        },
                      ]
        }
        data={
                Array [
                        Object {
                          "city": "San Diego",
                          "date": "04-05-2018",
                          "id": 1577,
                          "raceName": "Spooky 5k",
                          "remove": <p
                            onClick={[Function]}
                >
                             X 
                </p>,
                          "state": "CA",
                        },
                      ]
        }
        defaultExpanded={Object {}}
        defaultFilterMethod={[Function]}
        defaultFiltered={Array []}
        defaultPageSize={20}
        defaultResized={Array []}
        defaultSortDesc={false}
        defaultSortMethod={[Function]}
        defaultSorted={Array []}
        expanderDefaults={
                Object {
                        "filterable": false,
                        "resizable": false,
                        "sortable": false,
                        "width": 35,
                      }
        }
        filterable={false}
        freezeWhenExpanded={false}
        getLoadingProps={[Function]}
        getNoDataProps={[Function]}
        getPaginationProps={[Function]}
        getProps={[Function]}
        getResizerProps={[Function]}
        getTableProps={[Function]}
        getTbodyProps={[Function]}
        getTdProps={[Function]}
        getTfootProps={[Function]}
        getTfootTdProps={[Function]}
        getTfootTrProps={[Function]}
        getTheadFilterProps={[Function]}
        getTheadFilterThProps={[Function]}
        getTheadFilterTrProps={[Function]}
        getTheadGroupProps={[Function]}
        getTheadGroupThProps={[Function]}
        getTheadGroupTrProps={[Function]}
        getTheadProps={[Function]}
        getTheadThProps={[Function]}
        getTheadTrProps={[Function]}
        getTrGroupProps={[Function]}
        getTrProps={[Function]}
        groupedByPivotKey="_groupedByPivot"
        indexKey="_index"
        loading={false}
        loadingText="Loading..."
        multiSort={true}
        nestingLevelKey="_nestingLevel"
        nextText="Next"
        noDataText="No rows found"
        ofText="of"
        onExpandedChange={undefined}
        onFetchData={[Function]}
        onFilteredChange={undefined}
        onPageChange={undefined}
        onPageSizeChange={undefined}
        onResizedChange={undefined}
        onSortedChange={undefined}
        originalKey="_original"
        pageSize={1}
        pageSizeOptions={
                Array [
                        5,
                        10,
                        20,
                        25,
                        50,
                        100,
                      ]
        }
        pageText="Page"
        pivotBy={undefined}
        pivotDefaults={Object {}}
        pivotIDKey="_pivotID"
        pivotValKey="_pivotVal"
        previousText="Previous"
        resizable={true}
        rowsText="rows"
        showPageJump={true}
        showPageSizeOptions={true}
        showPagination={true}
        showPaginationBottom={true}
        showPaginationTop={false}
        sortable={true}
        style={Object {}}
        subRowsKey="_subRows"
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "AggregatedComponent": [Function],
        "ExpanderComponent": [Function],
        "FilterComponent": [Function],
        "LoadingComponent": [Function],
        "NextComponent": undefined,
        "NoDataComponent": [Function],
        "PadRowComponent": [Function],
        "PaginationComponent": [Function],
        "PivotComponent": undefined,
        "PivotValueComponent": [Function],
        "PreviousComponent": undefined,
        "ResizerComponent": [Function],
        "TableComponent": [Function],
        "TbodyComponent": [Function],
        "TdComponent": [Function],
        "TfootComponent": [Function],
        "ThComponent": [Function],
        "TheadComponent": [Function],
        "TrComponent": [Function],
        "TrGroupComponent": [Function],
        "aggregatedKey": "_aggregated",
        "className": "",
        "collapseOnDataChange": true,
        "collapseOnPageChange": true,
        "collapseOnSortingChange": true,
        "column": Object {
          "Aggregated": undefined,
          "Cell": undefined,
          "Expander": undefined,
          "Filter": undefined,
          "Footer": undefined,
          "Header": undefined,
          "Pivot": undefined,
          "PivotValue": undefined,
          "aggregate": undefined,
          "className": "",
          "filterAll": false,
          "filterMethod": undefined,
          "filterable": undefined,
          "footerClassName": "",
          "footerStyle": Object {},
          "getFooterProps": [Function],
          "getHeaderProps": [Function],
          "getProps": [Function],
          "headerClassName": "",
          "headerStyle": Object {},
          "minWidth": 100,
          "resizable": undefined,
          "show": true,
          "sortMethod": undefined,
          "sortable": undefined,
          "style": Object {},
        },
        "columns": Array [
          Object {
            "Header": "Race",
            "accessor": "raceName",
          },
          Object {
            "Header": "Distance",
            "accessor": "distance",
          },
          Object {
            "Header": "Time",
            "accessor": "time",
          },
          Object {
            "Header": "City",
            "accessor": "city",
          },
          Object {
            "Header": "State",
            "accessor": "state",
          },
          Object {
            "Header": "Date",
            "accessor": "date",
          },
          Object {
            "Header": "Completed?",
            "accessor": "completed",
          },
          Object {
            "Header": "Remove",
            "accessor": "remove",
          },
        ],
        "data": Array [
          Object {
            "city": "San Diego",
            "date": "04-05-2018",
            "id": 1577,
            "raceName": "Spooky 5k",
            "remove": <p
              onClick={[Function]}
>
               X 
</p>,
            "state": "CA",
          },
        ],
        "defaultExpanded": Object {},
        "defaultFilterMethod": [Function],
        "defaultFiltered": Array [],
        "defaultPageSize": 20,
        "defaultResized": Array [],
        "defaultSortDesc": false,
        "defaultSortMethod": [Function],
        "defaultSorted": Array [],
        "expanderDefaults": Object {
          "filterable": false,
          "resizable": false,
          "sortable": false,
          "width": 35,
        },
        "filterable": false,
        "freezeWhenExpanded": false,
        "getLoadingProps": [Function],
        "getNoDataProps": [Function],
        "getPaginationProps": [Function],
        "getProps": [Function],
        "getResizerProps": [Function],
        "getTableProps": [Function],
        "getTbodyProps": [Function],
        "getTdProps": [Function],
        "getTfootProps": [Function],
        "getTfootTdProps": [Function],
        "getTfootTrProps": [Function],
        "getTheadFilterProps": [Function],
        "getTheadFilterThProps": [Function],
        "getTheadFilterTrProps": [Function],
        "getTheadGroupProps": [Function],
        "getTheadGroupThProps": [Function],
        "getTheadGroupTrProps": [Function],
        "getTheadProps": [Function],
        "getTheadThProps": [Function],
        "getTheadTrProps": [Function],
        "getTrGroupProps": [Function],
        "getTrProps": [Function],
        "groupedByPivotKey": "_groupedByPivot",
        "indexKey": "_index",
        "loading": false,
        "loadingText": "Loading...",
        "multiSort": true,
        "nestingLevelKey": "_nestingLevel",
        "nextText": "Next",
        "noDataText": "No rows found",
        "ofText": "of",
        "onExpandedChange": undefined,
        "onFetchData": [Function],
        "onFilteredChange": undefined,
        "onPageChange": undefined,
        "onPageSizeChange": undefined,
        "onResizedChange": undefined,
        "onSortedChange": undefined,
        "originalKey": "_original",
        "pageSize": 1,
        "pageSizeOptions": Array [
          5,
          10,
          20,
          25,
          50,
          100,
        ],
        "pageText": "Page",
        "pivotBy": undefined,
        "pivotDefaults": Object {},
        "pivotIDKey": "_pivotID",
        "pivotValKey": "_pivotVal",
        "previousText": "Previous",
        "resizable": true,
        "rowsText": "rows",
        "showPageJump": true,
        "showPageSizeOptions": true,
        "showPagination": true,
        "showPaginationBottom": true,
        "showPaginationTop": false,
        "sortable": true,
        "style": Object {},
        "subRowsKey": "_subRows",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ReactTable
          AggregatedComponent={[Function]}
          ExpanderComponent={[Function]}
          FilterComponent={[Function]}
          LoadingComponent={[Function]}
          NextComponent={undefined}
          NoDataComponent={[Function]}
          PadRowComponent={[Function]}
          PaginationComponent={[Function]}
          PivotComponent={undefined}
          PivotValueComponent={[Function]}
          PreviousComponent={undefined}
          ResizerComponent={[Function]}
          TableComponent={[Function]}
          TbodyComponent={[Function]}
          TdComponent={[Function]}
          TfootComponent={[Function]}
          ThComponent={[Function]}
          TheadComponent={[Function]}
          TrComponent={[Function]}
          TrGroupComponent={[Function]}
          aggregatedKey="_aggregated"
          className=""
          collapseOnDataChange={true}
          collapseOnPageChange={true}
          collapseOnSortingChange={true}
          column={
                    Object {
                              "Aggregated": undefined,
                              "Cell": undefined,
                              "Expander": undefined,
                              "Filter": undefined,
                              "Footer": undefined,
                              "Header": undefined,
                              "Pivot": undefined,
                              "PivotValue": undefined,
                              "aggregate": undefined,
                              "className": "",
                              "filterAll": false,
                              "filterMethod": undefined,
                              "filterable": undefined,
                              "footerClassName": "",
                              "footerStyle": Object {},
                              "getFooterProps": [Function],
                              "getHeaderProps": [Function],
                              "getProps": [Function],
                              "headerClassName": "",
                              "headerStyle": Object {},
                              "minWidth": 100,
                              "resizable": undefined,
                              "show": true,
                              "sortMethod": undefined,
                              "sortable": undefined,
                              "style": Object {},
                            }
          }
          columns={
                    Array [
                              Object {
                                "Header": "Race",
                                "accessor": "raceName",
                              },
                              Object {
                                "Header": "Distance",
                                "accessor": "distance",
                              },
                              Object {
                                "Header": "Time",
                                "accessor": "time",
                              },
                              Object {
                                "Header": "City",
                                "accessor": "city",
                              },
                              Object {
                                "Header": "State",
                                "accessor": "state",
                              },
                              Object {
                                "Header": "Date",
                                "accessor": "date",
                              },
                              Object {
                                "Header": "Completed?",
                                "accessor": "completed",
                              },
                              Object {
                                "Header": "Remove",
                                "accessor": "remove",
                              },
                            ]
          }
          data={
                    Array [
                              Object {
                                "city": "San Diego",
                                "date": "04-05-2018",
                                "id": 1577,
                                "raceName": "Spooky 5k",
                                "remove": <p
                                  onClick={[Function]}
                    >
                                   X 
                    </p>,
                                "state": "CA",
                              },
                            ]
          }
          defaultExpanded={Object {}}
          defaultFilterMethod={[Function]}
          defaultFiltered={Array []}
          defaultPageSize={20}
          defaultResized={Array []}
          defaultSortDesc={false}
          defaultSortMethod={[Function]}
          defaultSorted={Array []}
          expanderDefaults={
                    Object {
                              "filterable": false,
                              "resizable": false,
                              "sortable": false,
                              "width": 35,
                            }
          }
          filterable={false}
          freezeWhenExpanded={false}
          getLoadingProps={[Function]}
          getNoDataProps={[Function]}
          getPaginationProps={[Function]}
          getProps={[Function]}
          getResizerProps={[Function]}
          getTableProps={[Function]}
          getTbodyProps={[Function]}
          getTdProps={[Function]}
          getTfootProps={[Function]}
          getTfootTdProps={[Function]}
          getTfootTrProps={[Function]}
          getTheadFilterProps={[Function]}
          getTheadFilterThProps={[Function]}
          getTheadFilterTrProps={[Function]}
          getTheadGroupProps={[Function]}
          getTheadGroupThProps={[Function]}
          getTheadGroupTrProps={[Function]}
          getTheadProps={[Function]}
          getTheadThProps={[Function]}
          getTheadTrProps={[Function]}
          getTrGroupProps={[Function]}
          getTrProps={[Function]}
          groupedByPivotKey="_groupedByPivot"
          indexKey="_index"
          loading={false}
          loadingText="Loading..."
          multiSort={true}
          nestingLevelKey="_nestingLevel"
          nextText="Next"
          noDataText="No rows found"
          ofText="of"
          onExpandedChange={undefined}
          onFetchData={[Function]}
          onFilteredChange={undefined}
          onPageChange={undefined}
          onPageSizeChange={undefined}
          onResizedChange={undefined}
          onSortedChange={undefined}
          originalKey="_original"
          pageSize={1}
          pageSizeOptions={
                    Array [
                              5,
                              10,
                              20,
                              25,
                              50,
                              100,
                            ]
          }
          pageText="Page"
          pivotBy={undefined}
          pivotDefaults={Object {}}
          pivotIDKey="_pivotID"
          pivotValKey="_pivotVal"
          previousText="Previous"
          resizable={true}
          rowsText="rows"
          showPageJump={true}
          showPageSizeOptions={true}
          showPagination={true}
          showPaginationBottom={true}
          showPaginationTop={false}
          sortable={true}
          style={Object {}}
          subRowsKey="_subRows"
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "AggregatedComponent": [Function],
          "ExpanderComponent": [Function],
          "FilterComponent": [Function],
          "LoadingComponent": [Function],
          "NextComponent": undefined,
          "NoDataComponent": [Function],
          "PadRowComponent": [Function],
          "PaginationComponent": [Function],
          "PivotComponent": undefined,
          "PivotValueComponent": [Function],
          "PreviousComponent": undefined,
          "ResizerComponent": [Function],
          "TableComponent": [Function],
          "TbodyComponent": [Function],
          "TdComponent": [Function],
          "TfootComponent": [Function],
          "ThComponent": [Function],
          "TheadComponent": [Function],
          "TrComponent": [Function],
          "TrGroupComponent": [Function],
          "aggregatedKey": "_aggregated",
          "className": "",
          "collapseOnDataChange": true,
          "collapseOnPageChange": true,
          "collapseOnSortingChange": true,
          "column": Object {
            "Aggregated": undefined,
            "Cell": undefined,
            "Expander": undefined,
            "Filter": undefined,
            "Footer": undefined,
            "Header": undefined,
            "Pivot": undefined,
            "PivotValue": undefined,
            "aggregate": undefined,
            "className": "",
            "filterAll": false,
            "filterMethod": undefined,
            "filterable": undefined,
            "footerClassName": "",
            "footerStyle": Object {},
            "getFooterProps": [Function],
            "getHeaderProps": [Function],
            "getProps": [Function],
            "headerClassName": "",
            "headerStyle": Object {},
            "minWidth": 100,
            "resizable": undefined,
            "show": true,
            "sortMethod": undefined,
            "sortable": undefined,
            "style": Object {},
          },
          "columns": Array [
            Object {
              "Header": "Race",
              "accessor": "raceName",
            },
            Object {
              "Header": "Distance",
              "accessor": "distance",
            },
            Object {
              "Header": "Time",
              "accessor": "time",
            },
            Object {
              "Header": "City",
              "accessor": "city",
            },
            Object {
              "Header": "State",
              "accessor": "state",
            },
            Object {
              "Header": "Date",
              "accessor": "date",
            },
            Object {
              "Header": "Completed?",
              "accessor": "completed",
            },
            Object {
              "Header": "Remove",
              "accessor": "remove",
            },
          ],
          "data": Array [
            Object {
              "city": "San Diego",
              "date": "04-05-2018",
              "id": 1577,
              "raceName": "Spooky 5k",
              "remove": <p
                onClick={[Function]}
>
                 X 
</p>,
              "state": "CA",
            },
          ],
          "defaultExpanded": Object {},
          "defaultFilterMethod": [Function],
          "defaultFiltered": Array [],
          "defaultPageSize": 20,
          "defaultResized": Array [],
          "defaultSortDesc": false,
          "defaultSortMethod": [Function],
          "defaultSorted": Array [],
          "expanderDefaults": Object {
            "filterable": false,
            "resizable": false,
            "sortable": false,
            "width": 35,
          },
          "filterable": false,
          "freezeWhenExpanded": false,
          "getLoadingProps": [Function],
          "getNoDataProps": [Function],
          "getPaginationProps": [Function],
          "getProps": [Function],
          "getResizerProps": [Function],
          "getTableProps": [Function],
          "getTbodyProps": [Function],
          "getTdProps": [Function],
          "getTfootProps": [Function],
          "getTfootTdProps": [Function],
          "getTfootTrProps": [Function],
          "getTheadFilterProps": [Function],
          "getTheadFilterThProps": [Function],
          "getTheadFilterTrProps": [Function],
          "getTheadGroupProps": [Function],
          "getTheadGroupThProps": [Function],
          "getTheadGroupTrProps": [Function],
          "getTheadProps": [Function],
          "getTheadThProps": [Function],
          "getTheadTrProps": [Function],
          "getTrGroupProps": [Function],
          "getTrProps": [Function],
          "groupedByPivotKey": "_groupedByPivot",
          "indexKey": "_index",
          "loading": false,
          "loadingText": "Loading...",
          "multiSort": true,
          "nestingLevelKey": "_nestingLevel",
          "nextText": "Next",
          "noDataText": "No rows found",
          "ofText": "of",
          "onExpandedChange": undefined,
          "onFetchData": [Function],
          "onFilteredChange": undefined,
          "onPageChange": undefined,
          "onPageSizeChange": undefined,
          "onResizedChange": undefined,
          "onSortedChange": undefined,
          "originalKey": "_original",
          "pageSize": 1,
          "pageSizeOptions": Array [
            5,
            10,
            20,
            25,
            50,
            100,
          ],
          "pageText": "Page",
          "pivotBy": undefined,
          "pivotDefaults": Object {},
          "pivotIDKey": "_pivotID",
          "pivotValKey": "_pivotVal",
          "previousText": "Previous",
          "resizable": true,
          "rowsText": "rows",
          "showPageJump": true,
          "showPageSizeOptions": true,
          "showPagination": true,
          "showPaginationBottom": true,
          "showPaginationTop": false,
          "sortable": true,
          "style": Object {},
          "subRowsKey": "_subRows",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
